code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1002,32 +1019,34 @@
 				if(H.lying)
 					atk_verb = "kick"
 
-				var/damage = rand(0, 9) + M.dna.species.punchmod
+				var/hitcheck = rand(0, 9)
+				var/damage = pick(0.5, 1, 1.5, 2) + M.dna.species.punchmod
 
 				var/obj/item/organ/limb/affecting = H.getrandomorgan(M.zone_sel.selecting)
 				var/armor_block = H.run_armor_check(affecting, "melee")
-				var/dmgcheck = H.apply_damage(damage, BRUTE, affecting, armor_block)
-				if(!damage || !istype(affecting) || !dmgcheck)
+				var/dmgcheck = apply_damage(damage, BRUTE, affecting, armor_block, H)
+
+				if(hitcheck == 0 || !dmgcheck)
 					playsound(H.loc, M.dna.species.miss_sound, 25, 1, -1)
 					H.visible_message("<span class='warning'>[M] has attempted to [atk_verb] [H]!</span>")
 					return 0
 
 				playsound(H.loc, get_sfx(M.dna.species.attack_sound), 25, 1, -1)
-
+				H.apply_damage(damage*3+2, STAMINA, affecting, armor_block)
```

Comment:
Damage*3 + 2? Does this mean a 30 brute force item would do 92 stamina, or am I missing something obvious?